README.md

```diff
@@ -133,7 +136,8 @@ formats or signing routines change.
 
 For a detailed list of all command line arguments and their usage, run
 `in-toto-sign --help` or look at the
-[code documentation](https://github.com/in-toto/in-toto/blob/develop/in_toto/in_toto_sign.py).
+[online
+documentation](https://in-toto.readthedocs.io/en/latest/command-line-tools/in-toto-sign.html).
```

Comment:
Is it me, or the line wrapping here also causes the links to be split? (or it could be the richdiff viewer being wonky as well..)

Reply:
AFAICS the markdown parser can deal line breaks inside brackets. The rendered README also suggests that that's the case: https://github.com/in-toto/in-toto/blob/5578d3e871171cc1f9bcce9f4228a58c2ab7ef27/README.md